src/core_plugins/console/public/src/directives/sense_settings.js

```diff
@@ -16,6 +16,16 @@ require('ui/modules')
         $scope.kbnTopNav.close();
       };
 
+      const self = this;
+
+      function onEnter(event) {
+        if (event.which === 13) {
+          self.apply();
+        }
+      }
```

Comment:
I wanted a named reference to the function so I could unbind it. If I'd used arrow syntax, I would have to assign it to a variable, which to me defeats the point of using an anonymous function.